python/py-pytest-env/Portfile

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pytest-env
+version             1.1.3
+revision            0
+
+categories-append   devel
+platforms           {darwin any}
+supported_archs     noarch
+license             MIT
+maintainers         nomaintainer
+
+description         pytest plugin to set environment variables
+
+long_description    {*}${description} in pytest.ini or pyproject.toml file
+
+homepage            https://github.com/pytest-dev/pytest-env
+
+distname            [string map {- _} ${python.rootname}]-${version}
+
+checksums           rmd160  d92e54bb1b755877683dc41b3a62d9b15ced3f30 \
+                    sha256  fcd7dc23bb71efd3d35632bde1bbe5ee8c8dc4489d6617fb010674880d96216b \
+                    size    8627
+
+python.versions     310 311 312
+python.pep517_backend \
+                    hatch
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-hatch-vcs
+    depends_run-append \
+                    port:py${python.version}-pytest
+
+    if {${python.version} < 311} {
+        depends_run-append \
+                    port:py${python.version}-tomli
+    }
+
+    pre-test {
+        test.env    PYTHONPATH=[glob -nocomplain ${destroot}/${python.pkgd}]
+    }
```

Comment:
is this "pre-test" needed - presumably the "python" PortGroup sets this already to an appropriate value.

Reply:
Déjà vu. You have rightfully given this note before. Then, and now, at some point _needed_ to add the PYTHONPATH for the test to work. Testing this again it works without. Wonder how that came to be?
I have now removed the lines.